src/test/java/com/sproutsocial/nsq/BackoffDockerTestIT.java

```diff
@@ -0,0 +1,84 @@
+package com.sproutsocial.nsq;
+
+import org.junit.Test;
```

Comment:
This is a NIT: may be we can upgrade to junit5. Can be done in a later change as well though.